crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -875,8 +875,6 @@ impl<'a, T: 'static> SystemParamFetch<'a> for NonSendMutState<T> {
     }
 }
 
-pub struct OrState<T>(T);
```

Comment:
I believe this is a left over from the ecs 2.0 rework, it doesn't seem to be used anymore so I figured it should be fine to remove it?

Reply:
I agree; I think this is just dead code.